No Check Annotation for compile error in C# project

I [introduced a simple compile error](https://github.com/rainersigwald/bumblebee/pull/6/commits/10122ffed905bc0408ecb1894380402e53c2b7b4), which seems to have been matched by the problem matcher (it turned red in the console output view and is prefixed with `##[error]`), but there was no annotation linking the error to the file/line number information that is contained in the line.

https://github.com/rainersigwald/bumblebee/pull/6/checks?check_run_id=192916618#step:4:32

I hoped for the sort of linking in the Files Changed tab of the PR that are described in [the Checks CI docs](https://developer.github.com/apps/quickstart-guides/creating-ci-tests-with-the-checks-api/#step-25-updating-the-check-run-with-ci-test-results).

This _might_ be because the error includes only a relative-to-the-project filename, rather than relative-to-the-repo-root?

--- a/.github/csc.json
+++ b/.github/csc.json
@@ -4,9 +4,9 @@
             "owner": "csc",
             "pattern": [
                 {
-                    "regexp": "^([^\\s].*)\\((\\d+|\\d+,\\d+|\\d+,\\d+,\\d+,\\d+)\\):\\s+(error|warning|info)\\s+(\\w+\\d+):\\s*(.*?)\\s+\\[(.*?)\\]$",
+                    "regexp": "^([^\s].*)\((\d+)(?:,\d+|,\d+,\d+)?\):\s+(error|warning)\s+([a-zA-Z]+(?<!MSB)\d+):\s*(.*?)\s+\[(.*?)\]$",
                     "file": 1,
-                    "location": 2,
+                    "line": 2,
                     "severity": 3,
                     "code": 4,
                     "message": 5,

--- a/.github/csc.json
+++ b/.github/csc.json
@@ -4,12 +4,13 @@
             "owner": "csc",
             "pattern": [
                 {
-                    "regexp": "^([^\\s].*)\\((\\d+|\\d+,\\d+|\\d+,\\d+,\\d+,\\d+)\\):\\s+(error|warning|info)\\s+(\\w+\\d+)\\s*:\\s*(.*)$",
+                    "regexp": "^([^\\s].*)\\((\\d+|\\d+,\\d+|\\d+,\\d+,\\d+,\\d+)\\):\\s+(error|warning|info)\\s+(\\w+\\d+):\\s*(.*?)\\s+\\[(.*?)\\]$",
                     "file": 1,
                     "location": 2,
                     "severity": 3,
                     "code": 4,
-                    "message": 5
+                    "message": 5,
+                    "fromPath": 6
                 }
             ]
         }

--- a/.github/csc.json
+++ b/.github/csc.json
@@ -4,7 +4,7 @@
             "owner": "csc",
             "pattern": [
                 {
-                    "regexp": "^([^\s].*)\((\d+)(?:,\d+|,\d+,\d+)?\):\s+(error|warning)\s+([a-zA-Z]+(?<!MSB)\d+):\s*(.*?)\s+\[(.*?)\]$",
+                    "regexp": "^([^\\s].*)\((\\d+)(?:,\\d+|,\\d+,\\d+)?\\):\\s+(error|warning)\\s+([a-zA-Z]+(?<!MSB)\\d+):\\s*(.*?)\\s+\\[(.*?)\\]$",
                     "file": 1,
                     "line": 2,
                     "severity": 3,

--- a/.github/csc.json
+++ b/.github/csc.json
@@ -4,7 +4,7 @@
             "owner": "csc",
             "pattern": [
                 {
-                    "regexp": "^([^\\s].*)\((\\d+)(?:,\\d+|,\\d+,\\d+)?\\):\\s+(error|warning)\\s+([a-zA-Z]+(?<!MSB)\\d+):\\s*(.*?)\\s+\\[(.*?)\\]$",
+                    "regexp": "^([^\\s].*)\\((\\d+)(?:,\\d+|,\\d+,\\d+)?\\):\\s+(error|warning)\\s+([a-zA-Z]+(?<!MSB)\\d+):\\s*(.*?)\\s+\\[(.*?)\\]$",
                     "file": 1,
                     "line": 2,
                     "severity": 3,

--- a/.github/csc.json
+++ b/.github/csc.json
@@ -4,7 +4,7 @@
             "owner": "csc",
             "pattern": [
                 {
-                    "regexp": "^([^\\s].*)\\((\\d+|\\d+,\\d+|\\d+,\\d+,\\d+,\\d+)\\):\\s+(error|warning|info)\\s+(CS\\d+)\\s*:\\s*(.*)$",
+                    "regexp": "^([^\\s].*)\\((\\d+|\\d+,\\d+|\\d+,\\d+,\\d+,\\d+)\\):\\s+(error|warning|info)\\s+(\\w+\\d+)\\s*:\\s*(.*)$",
                     "file": 1,
                     "location": 2,
                     "severity": 3,

